test/test_unistd.rs

```diff
@@ -400,3 +403,55 @@ fn test_pipe2() {
     let f1 = FdFlag::from_bits_truncate(fcntl(fd1, FcntlArg::F_GETFD).unwrap());
     assert!(f1.contains(FdFlag::FD_CLOEXEC));
 }
+
+// Used in `test_alarm`.
+static mut ALARM_CALLED: bool = false;
+
+// Used in `test_alarm`.
+#[no_mangle]
```

Comment:
What's the purpose of `no_mangle`?  Using `extern` should be sufficient to pass the function pointer to a C function.

Reply:
I just copied a handle from my code, which (if I can remeber correctly) needed this. Will see if I can remove it.

Reply:
You're right it wasn't needed.